.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+
+
+
+<!-- Describe your PR here. -->
+
+
+
+<!--
+
+  The following applies to third-party contributors.
+  Sentry employees and contractors can delete or ignore.
+
+-->
+
+----
+
+By submitting this pull request, I confirm that Sentry can use, modify, copy, and redistribute this contribution, under Sentry's choice of terms.
```

Comment:
```suggestion
By submitting this pull request, I acknowledge that I love Sentry by letting them use, modify, copy, and redistribute my organic and artisanal code that does not violate anyone else's rights without any limitations. I also acknowledge that this is some boring legalese but my love of Sentry trumps anything.
```

Reply:
@chadwhitacre 
>Still need to give the verbiage that Sentry flair tho ... ;)

How about something like this? :)

Reply:
loving @BYK 's input here